src/vuestic-theme/vuestic-sass/resources/_variables.scss

```diff
@@ -177,6 +177,7 @@ $btn-padding-y-lg: 0.75rem;
 $btn-padding-x-lg: 1.75rem;
 $btn-font-size-lg: 1.25rem;
 $btn-line-height-lg: 1.3;
+$btn-line-height-lg-outline: 1.05; // = $btn-line-height-lg - 2 * $btn-border-outline
```

Comment:
Can't you derive this in scss instead of computing on your own?

Reply:
Would like to use `calc()`, but it doesn't seem to handle these variables :confused: